### PR TITLE
Include cstdint to fix build on recent gcc toolchain. (Tested on gcc 13)

### PR DESCRIPTION
**Pull request by sbourasse**
I guess the title should be clear enough.
If you have any questions or concerns let me know.

### CODE REVIEW

**Review by cary-ilm — APPROVED** [review]
LGTM

**Comment by cary-ilm** [comment]
Thanks for the catch, looks fine.

Our project policy is to require a signed commits and a contributor license agreement on all PRs. Would you kindly click on the "details" button next to the DCO and EasyCLA checks above and step through the process?

File: src/lib/OpenEXR/ImfDeepTiledInputFile.h [hunk 1]
```diff
@@ -22,6 +22,8 @@
 #include <cstdint>
 #include <ImathBox.h>
```
Comment (kmilos) [inline src/lib/OpenEXR/ImfDeepTiledInputFile.h:24]: Already included a few lines above?

File: src/lib/OpenEXR/ImfDeepTiledInputPart.h [hunk 2]
```diff
@@ -13,6 +13,8 @@
 #include <cstdint>
 #include <ImathBox.h>
 
+#include <cstdint>
```
Comment (kmilos) [inline src/lib/OpenEXR/ImfDeepTiledInputPart.h:16]: Ditto

**Review by kmilos — COMMENTED** [review]
Strange - OpenEXR already builds just fine w/ GCC 13 on e.g. MSYS2 UCRT64...

**Comment by kmilos** [comment]
See https://github.com/AcademySoftwareFoundation/openexr/commit/c5763cfdf59ef4880dacdb11664782822a0976cd and https://github.com/AcademySoftwareFoundation/openexr/commit/cc508f7eddbf1290c61daee69dc247c3fc6c291f

**Comment by sbourasse** [comment]
Yep, we fixed this on our side, and I didn't realise when rebasing that the update had already been done.
Apologies, I'll close.

**Pull request closed without merging**